src/App.js

```diff
@@ -13,10 +23,149 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods 1</h1>
-  </div>
-);
+class App extends React.Component {
+  state = {
+    showList: false,
+    copyList: [...goodsFromServer],
+    selectValue: 0,
+  }
+
+  getSelectValue = event => (
+    this.setState({
+      selectValue: event.target.value,
+      copyList: [...goodsFromServer]
+        .filter(elem => elem.length === Number(event.target.value)),
+    })
+  );
+
+  toShowList = () => (
```

Comment:
what does `toShowList` mean?
what does this function do?

Reply:
It show my Start button and then show my another button and list

Reply:
handleStart